### PR TITLE
[81] added enum

### PR DESCRIPTION
**Pull request by ozhurova**
added enum
image

### CODE REVIEW

File: src/app/enums/enums.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export enum EnglishLevels {
```
Comment (rastorc3v) [inline src/app/enums/enums.ts:1]: rename file to english-levels.enum.ts or something like that

**Review by rastorc3v — CHANGES_REQUESTED** [review]
attach screenshot please

File: src/app/enums/enums.ts (hunk 1 above)
Comment (ozhurova) [reply src/app/enums/enums.ts]: done

**Comment by ozhurova** [comment]
image

**Review by mhasilau — APPROVED** [review]

**Review by Nicolai-boiko — APPROVED** [review]

**Review by vstrabko — APPROVED** [review]

**Review by rastorc3v — APPROVED** [review]

**Pull request merged**